src/core/meshcop/commissioner.hpp

```diff
@@ -157,8 +161,9 @@ class Commissioner : public InstanceLocator
      * @param[in]  aTlvs        A pointer to Commissioning Data TLVs.
      * @param[in]  aLength      The length of requested TLVs in bytes.
      *
-     * @retval OT_ERROR_NONE     Send MGMT_COMMISSIONER_GET successfully.
-     * @retval OT_ERROR_FAILED   Send MGMT_COMMISSIONER_GET fail.
+     * @retval OT_ERROR_NONE           Send MGMT_COMMISSIONER_GET successfully.
+     * @retval OT_ERROR_FAILED         Send MGMT_COMMISSIONER_GET fail.
```

Comment:
Can this return `OT_ERROR_FAILED`?  I believe it can also return `OT_ERROR_NO_BUFS`.